horbatiuk-vitalii/task_03/script_Linux_OS.sh

```
@@ -0,0 +1,31 @@
+LOG_FILE="/my_files/backup.log"
+
+SOURCE="$1"
+DESTINATION="$2"
+SIZE_THRESHOLD="$3"
+REMOTE_HOST="root@172.30.2.2"
+
+echo "$(date): Start copying..." | tee -a "$LOG_FILE"
+
+EXCLUDED_FOLDERS=$(ssh "$REMOTE_HOST" "find '$DESTINATION' -type d -exec du -s {} + | awk '\$1 > $SIZE_THR>
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)